src/main/java/org/sonarqube/A.java

```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
+    message.length();
+
     return message;
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') BUG
A "NullPointerException" could be thrown; "message" is nullable here.(AWBEzrmbI9pSNYZJmpeH)

[ISSUE_KEY]: AWBEzrmbI9pSNYZJmpeH